src/main/perf/LocalTaskSource.java

```diff
@@ -166,7 +168,8 @@ static List<Task> loadTasks(TaskParser taskParser, String filePath) throws IOExc
         continue;
       }
 
-      tasks.add(taskParser.parseOneTask(line));
+      // Only parse the category here, will parse to specific task when searching
+      tasks.add(new UnparsedTask(parseCategory(line)));
```

Comment:
Hmm, so this will make all tasks always late binding (re-parse every time it executes)?

Maybe we should make this part of the `TaskParser` syntax itself?  This way the benchmark could choose which tasks to "parse early" and which to "parse late"?

Reply:
Yeah kind of, in new commit I made 2 methods `firstPassParse` and `secondPassParse` where `firstPassParse` was doing this line and secondPassParse is called within `UnparsedTask`.
But still I default to late-binding any task -- any idea on which task should late binding and which should not?